### ISSUE TITLE
escaped ** in comments?

### ISSUE THREAD

**Issue by kazamatzuri** [PagerDuty/pdpyras#125]
When pdpyras is being imported as part of a unittest via importlib._bootstrap, we're getting an error: 

```


/pyenv/versions/3.10.8/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:177: in exec_module
--
  | source_stat, co = _rewrite_test(fn, self.config)
  | /pyenv/versions/3.10.8/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:359: in _rewrite_test
  | tree = ast.parse(source, filename=strfn)
  | /pyenv/versions/3.10.8/lib/python3.10/ast.py:50: in parse
  | return compile(source, filename, mode, flags,
  | E     File "/pyenv/versions/3.10.8/lib/python3.10/site-packages/pdpyras.py", line 1078
  | E       """
  | E       ^^^
  | E   SyntaxError: invalid escape sequence '\*'

```

This is due to 
in pdpyras.py:1078
```
:param \*\*kwargs:
```

Why are the * escaped there?

**Comment by Deconstrained** [comment]
Thanks for pointing this out. The original reason for that escape sequence was because `*` is a special character in Restructured Text (how Sphinx interprets docstrings) i.e. `*emphasis*` = _emphasis_.

However, I think the double-splat operator renders correctly without escaping in other parts of the codebase, so I'll give it a once-over and make them all consistent. It wasn't originally this way as I recall, but things seem to have changed.

**Comment by Deconstrained** [comment]
It produces warnings in Sphinx but Sphinx renders them correctly, so I'll make a new release to fix this.

**Comment by kazamatzuri** [comment]
Awesome, thank you!

**Comment by Deconstrained** [comment]
Fixed in v5.1.3
